lib/ui/views/liquidity_add/layouts/components/liquidity_add_confirm_sheet.dart

```diff
@@ -75,15 +81,50 @@ class LiquidityAddConfirmSheetState
             labelBtn: AppLocalizations.of(context)!.btn_confirm_liquidity_add,
             disabled: !consentChecked && liquidityAdd.consentDateTime == null,
             onPressed: () async {
+              if (liquidityAdd.token1 != null &&
+                  liquidityAdd.token1!.isUCO &&
+                  (Decimal.parse(
+                                liquidityAdd.token1Balance.toString(),
+                              ) -
+                              Decimal.parse(
+                                liquidityAdd.token1Amount.toString(),
+                              ))
+                          .toDouble() <
+                      lowUCOInDollarsWarningValue) {
+                final result = await LowUCOWarningPopup.getDialog(context);
+                if (result != null && result == false) {
+                  return;
+                }
+              }
+              if (liquidityAdd.token2 != null &&
+                  liquidityAdd.token2!.isUCO &&
+                  (Decimal.parse(
+                                liquidityAdd.token2Balance.toString(),
+                              ) -
+                              Decimal.parse(
+                                liquidityAdd.token2Amount.toString(),
+                              ))
+                          .toDouble() <
+                      lowUCOInDollarsWarningValue) {
+                if (context.mounted) {
+                  final result = await LowUCOWarningPopup.getDialog(context);
+                  if (result != null && result == false) {
+                    return;
+                  }
+                }
+              }
+
               final liquidityAddNotifier =
                   ref.read(liquidityAddFormNotifierProvider.notifier);
-              unawaited(
-                liquidityAddNotifier.add(AppLocalizations.of(context)!),
-              );
-              await LiquidityAddInProgressPopup.getDialog(
-                context,
-                ref,
-              );
+              if (context.mounted) {
```

Comment:
Shouldn't we check for `context.mounted` before reading the `liquidityAddFormNotifierProvider` ?